apps/mobile-wallet/src/components/NFTsGrid.tsx

```diff
@@ -17,33 +17,35 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { AddressHash, NFT } from '@alephium/shared'
-import { FlashList } from '@shopify/flash-list'
+import { FlashList, FlashListProps } from '@shopify/flash-list'
 import { useMemo } from 'react'
-import { ActivityIndicator, Dimensions } from 'react-native'
+import { useTranslation } from 'react-i18next'
+import { ActivityIndicator, Dimensions, NativeScrollEvent, NativeSyntheticEvent } from 'react-native'
 import styled, { useTheme } from 'styled-components/native'
 
 import AppText from '~/components/AppText'
 import NFTThumbnail from '~/components/NFTThumbnail'
-import { ModalContentProps } from '~/features/modals/ModalContent'
 import { useAppSelector } from '~/hooks/redux'
 import { makeSelectAddressesNFTs } from '~/store/addressesSlice'
 import { DEFAULT_MARGIN } from '~/style/globalStyle'
 
-interface NFTsGridProps extends ModalContentProps {
+interface NFTsGridProps extends Omit<Partial<FlashListProps<unknown>>, 'contentContainerStyle'> {
```

Comment:
But here we are using FlashList, not FlatList.

Reply:
So the problem here appears to be that the `Content` property of `BottomModalProps` is a function where its argument is of type `ModalContentProps` which extends `ScrollViewProps` which includes 
```ts
contentContainerStyle?: StyleProp<ViewStyle> | undefined
```

meaning that our typings are not very good here because the bottom modal expends a `ScrolScreen` (or something that extends it, like a `FlatList`) as its content but we are feeding it a `FlashList` here.

I agree with you that it is not worth the effort to fix this properly since we currently don't need to override the container styles. I would, however, add a comment above the `Omit` to explain why we are doing this because it is not clear at all when reading the code. 🙏